engine/step/step.go

```diff
@@ -103,6 +103,9 @@ type Step struct {
 	Item            interface{}     `json:"item,omitempty"` // "child" step: item value, issued from foreach
 
 	Resources []string `json:"resources"` // resource limits to enforce
+
+	NewTags     map[string]string `json:"new_tags"`
```

Comment:
Squash both into "ModifiedTags" ?

Reply:
Depends on the decision regarding splitting add and remove operations on tags.